config.yml

```diff
@@ -0,0 +1,9 @@
+---
+user_api:
+  host: users_api
+  port: 8080
```

Comment:
you don't need to split destination to **name** and **port**.
you may use **addr** with mixed value, **users_api:8080**

Reply:
done